cyrasterize/c_opengl_debug.pyx

```diff
@@ -14,15 +13,15 @@ class bcolors:
     END = '\x1b[0m'
 
 def printWarn(string, color=bcolors.YELLOW):
-    log.debug(color + string + bcolors.END)
+    log.debug(color + string + color)
```

Comment:
does this print the warning in a color (e.g. red?). Just getting my head around the color being either side (thinking like HTML tags?)